concent_api/core/message_handlers.py

```diff
@@ -822,14 +822,20 @@ def store_subtask(
     provider  = Client.objects.get_or_create_full_clean(provider_public_key)
     requestor = Client.objects.get_or_create_full_clean(requestor_public_key)
 
+    computation_deadline = task_to_compute.compute_task_def['deadline']
+
+    result_package_size = report_computed_task.size
+
     subtask = Subtask(
-        task_id         = task_id,
-        subtask_id      = subtask_id,
-        provider        = provider,
-        requestor       = requestor,
-        state           = state.name,
-        next_deadline   = parse_timestamp_to_utc_datetime(next_deadline) if next_deadline is not None else None,
-        task_to_compute=store_message(task_to_compute, task_id, subtask_id),
+        task_id             = task_id,
+        subtask_id          = subtask_id,
+        provider            = provider,
+        requestor           = requestor,
+        result_package_size = result_package_size,
+        state               = state.name,
+        next_deadline       = parse_timestamp_to_utc_datetime(next_deadline) if next_deadline is not None else None,
+        computation_deadline= parse_timestamp_to_utc_datetime(computation_deadline),
+        task_to_compute     =store_message(task_to_compute, task_id, subtask_id),
```

Comment:
Please fix styling.

Reply:
Done.

Reply:
Still there are some spaces on left side of `=`.

Reply:
Done.